src/Microsoft.ML.StandardTrainers/Standard/ModelStatistics.cs

```diff
@@ -16,249 +16,343 @@
-            : this(env, trainingExampleCount, paramCount, deviance, nullDeviance)
+        void ICanSaveModel.Save(ModelSaveContext ctx)
         {
-            _env.Assert(coeffStdError.GetValues().Length == _paramCount);
-            _coeffStdError = coeffStdError;
+            Contracts.AssertValue(Env);
+            Env.CheckValue(ctx, nameof(ctx));
+            SaveCore(ctx);
+            ctx.SetVersionInfo(GetVersionInfo());
         }
 
-        internal LinearModelStatistics(IHostEnvironment env, ModelLoadContext ctx)
+        protected virtual void SaveCore(ModelSaveContext ctx)
         {
-            Contracts.CheckValue(env, nameof(env));
-            _env = env;
-            _env.AssertValue(ctx);
-
             // *** Binary Format ***
             // int: count of parameters
             // long: count of training examples
             // float: deviance
             // float: null deviance
-            // bool: whether standard error is included
-            // (Conditional) float[_paramCount]: values of std errors of coefficients
-            // (Conditional) int: length of std errors of coefficients
-            // (Conditional) int[_paramCount]: indices of std errors of coefficients
 
-            _paramCount = ctx.Reader.ReadInt32();
-            _env.CheckDecode(_paramCount > 0);
+            Env.Assert(ParametersCount > 0);
+            ctx.Writer.Write(ParametersCount);
 
-            _trainingExampleCount = ctx.Reader.ReadInt64();
-            _env.CheckDecode(_trainingExampleCount > 0);
+            Env.Assert(TrainingExampleCount > 0);
+            ctx.Writer.Write(TrainingExampleCount);
 
-            _deviance = ctx.Reader.ReadFloat();
-            _nullDeviance = ctx.Reader.ReadFloat();
+            ctx.Writer.Write(Deviance);
+            ctx.Writer.Write(NullDeviance);
+        }
+
+        internal virtual void SaveText(TextWriter writer, DataViewSchema.Column featureColumn, int paramCountCap)
+        {
+            Contracts.AssertValue(Env);
+            Env.CheckValue(writer, nameof(writer));
+
+            writer.WriteLine();
+            writer.WriteLine("*** MODEL STATISTICS SUMMARY ***   ");
+            writer.WriteLine("Count of training examples:\t{0}", TrainingExampleCount);
+            writer.WriteLine("Residual Deviance:         \t{0}", Deviance);
+            writer.WriteLine("Null Deviance:             \t{0}", NullDeviance);
+            writer.WriteLine("AIC:                       \t{0}", 2 * ParametersCount + Deviance);
+        }
+
+        /// <summary>
+        /// Support method for linear models and <see cref="ICanGetSummaryInKeyValuePairs"/>.
+        /// </summary>
+        internal virtual void SaveSummaryInKeyValuePairs(DataViewSchema.Column featureColumn, int paramCountCap, List<KeyValuePair<string, object>> resultCollection)
```

Comment:
>featureColumn [](start = 79, length = 13)

Note this `featureColumn` parameter. The method `featureColumn.GetSlotNames` may be useful to get the feature names for expanding the report.